libs/components/core/src/lib/modules/overlay/overlay.component.ts

```diff
@@ -179,7 +181,17 @@ export class SkyOverlayComponent implements OnInit, OnDestroy {
     this.targetRef.clear();
 
     const injector = Injector.create({
-      providers,
+      providers: [
+        {
+          provide: SKY_STACKING_CONTEXT,
+          useValue: {
+            zIndex: new BehaviorSubject(parseInt(this.zIndex, 10)).pipe(
```

Comment:
This should be set to an `Observable<number>`, yes? Maybe return `asObservable()`?